### PR TITLE
adding the Back-End Lang and Tech to READEME.md #9

### PR DESCRIPTION
**Pull request by BozhidarLeshev**
Added the enumeration of the frameworks and database used the backend Languages and Technology section of the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,8 @@
 - <b>React.js</b> (JavaScript Library)
 - <b>SASS</b> (Syntactically Awesome Style Sheets - Preprocessor Scripting Language)
 - <b>Styled-Components</b> (Library for React.js)
+
+<b>Back-End:</b>
+- <b>Node.js</b>
+- <b>Express.js (Framework for Node.js)</b>
```
Comment (Zain-MKhan) [inline README.md:25]: Looks great, just remove the bolding for the text in parenthesis (Framework for Node.js)

Comment (BozhidarLeshev) [reply README.md:25]: Perfect, will change.

Comment (BozhidarLeshev) [reply README.md]: done

**Review by Zain-MKhan — APPROVED** [review]
Looks great!

**Pull request merged**